### ISSUE TITLE
ELB audit flapping due to changes in policies

### ISSUE THREAD

**Issue by AlexClineBB** [Netflix/security_monkey#262]
The ELB auditing is notifying of changes to all of our ELBs due to the following "change" seen below with different results at two times this morning:

```
Nov 20, 2015 7:58:30 AM
      "policies": [
        {
          "reference_security_policy": "ELBSecurityPolicy-2015-05",
          "type": "SSLNegotiationPolicyType",
          "name": "ELBSecurityPolicy-2015-05"
        }
      ],
```

```
Nov 20, 2015 11:59:16 AM
      "policies": [
        {
          "name": "ELBSecurityPolicy-2015-05"
        }
      ],
```

I'm not able to see the `reference_security_policy` or `type` attributes when I request the ELB config using the AWS API. No other changes to the ELBs are occurring.

**Comment by monkeysecurity** [comment]
I've seen this too.  I'll prioritize this bug.  I find it super annoying.

@AlexClineBB - Are other apps in your account hitting the ELB API's especially hard?

**Comment by AlexClineBB** [comment]
@monkeysecurity, yes we are using the ELB APIs pretty heavily - for instance attaching and configuration checking.

**Comment by monkeysecurity** [comment]
Could you see if the current develop branch exhibits this same behavior?  It better handles rate limiting in the botocore calls and doesn't give up as easily, which may help fix the problem.  I'm testing the develop branch as well and will update this ticket with what I find.

**Comment by monkeysecurity** [comment]
Fixed in #265 

This has been merged to master in release v0.4.0

**Comment by AlexClineBB** [comment]
@monkeysecurity I've deployed v0.4.0 internally and can see a big improvement in the number of messages being sent. Thanks for the quick fix!
